hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -1574,6 +1574,17 @@ public long prepareOzoneManager(
     return prepareResponse.getTxnID();
   }
 
+  public PrepareStatusResponse getOzoneManagerPrepareStatus(long txnId)
```

Comment:
Can you add the @Override here that will let us know this is a protocol method?

Reply:
Fixed.